core/src/edu/cornell/gdiac/temporary/entity/Note.java

```diff
@@ -17,13 +17,21 @@
  * Model class for Notes.
  */
 public class Note{
-	public static final float descentSpeed = -4.5f;
 	/** Rescale the size of a shell */
 	private static final float SHELL_SIZE_MULTIPLE = 4.0f;
 	/** How fast we change frames (one frame per 4 calls to update) */
 	private static final float ANIMATION_SPEED = 0.25f;
+
+	public int getNUM_ANIM_FRAMES() {
+		return NUM_ANIM_FRAMES;
+	}
+
+	public void setNUM_ANIM_FRAMES(int NUM_ANIM_FRAMES) {
+		this.NUM_ANIM_FRAMES = NUM_ANIM_FRAMES;
+	}
```

Comment:
why is this necessary? I don't see you using it anywhere (and I don't think it should be able to be modified, even by the BandMember class)